nightfall/findings.py

```diff
@@ -31,21 +31,25 @@ class Finding:
         byte_range (Range): The byte range in which a finding was detected within the item.
         codepoint_range (Range): The codepoint range in which a finding was detected within the item. This differs
             from byte range since a codepoint may contain multiple bytes.
+        row_range (Range): The row in which a finding was detected, if it was in a tabular document. Index starts at 1.
+        column_range (Range): The column(s) in which a finding was detected, if it was in a tabular document. Index starts at 1.
         commit_hash (str): The hash of the commit in which the finding was detected, if known.
         matched_detection_rule_uuids (List[str]): The list of detection rule UUIDs that contained a detector that
             triggered a match.
         matched_detection_rules (List[str]): The list of inline detection rules that contained a detector that triggered
             a match.
     """
     finding: str
-    redacted_finding: str
-    before_context: str
-    after_context: str
-    detector_name: str
+    redacted_finding: Optional[str]
+    before_context: Optional[str]
+    after_context: Optional[str]
+    detector_name: Optional[str]
```

Comment:
why are these fields changing to `Optional`?

Reply:
because they can be `None`. This could be a separate PR (not related to this work), but if you read the doc comment it says they can be `None`.